parts/k8s/windowsnodereset.ps1

```diff
@@ -83,6 +89,12 @@ if ($global:NetworkPlugin -eq 'kubenet') {
 #
 # Start Services
 #
+
+if ($global:CsiProxyEnabled) {
+    Write-Log "Starting csi-proxy-server service"
```

Comment:
I was thinking we would also be introducing service dependancy ? Would the start-service do sync start or would it do a start asynchronously potentially causing race scenarios with start of kubelet ?

Reply:
Oh yea - forgot we had discussed this.
I will add one.